client/state/index.js

```diff
@@ -58,6 +58,7 @@ let middleware = [ thunkMiddleware ];
 if ( typeof window === 'object' ) {
 	middleware = [
 		...middleware,
+		require( './analytics/middleware.js' ).analyticsMiddleware,
 		require( './themes/middlewares.js' ).analyticsMiddleware
```

Comment:
My goal here is probably to eliminate the themes middleware, as it's fairly specific and could be replaced with this one. That was the inspiration for my solution though, at least with regards to `redux-analytics`